src/constants.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  'PROXY_STATUS_WORKING': 'working',
```

Comment:
Maybe this might be worth using [Symbols](https://hacks.mozilla.org/2015/06/es6-in-depth-symbols/) for?